Upgrade to 1.6.7 step 2

When I try to upgrade to the new version 1.6.7 (from 1.6.7.rc6) I get an error while step 2:

```
unzip -t -q print/WEB-INF/lib/jasperreports-functions-5.6.1.jar
make: unzip: Command not found
make: *** [print/WEB-INF/lib/jasperreports-functions-5.6.1.jar] Error 127

```
Any idea what I could do?

--- a/c2cgeoportal/scaffolds/update/CONST_packages.yaml
+++ b/c2cgeoportal/scaffolds/update/CONST_packages.yaml
@@ -37,6 +37,9 @@ main:
     gettext:
         cmd: gettext -V | head -n1 | awk '{{print $4}}'
         version: 0.18
+    unzip:
+        cmd: unzip -v | head -n1 | awk '{{print $2}}'
+        version: 6
 
 mobile:
     sencha:

